kong/utils_test.go

```diff
@@ -1671,3 +2079,184 @@ func Test_FillPluginsDefaults_SetType(t *testing.T) {
 		})
 	}
 }
+
+func Test_FillPluginsDefaults_Acme(t *testing.T) {
+	RunWhenKong(t, "==3.3.0")
```

Comment:
Is there a reason we want to run this against 3.3 **only**? If so then I believe this line deserves a comment why that's the case. It's not obvious to the reader why this particular version was chosen here and deserves a special treatment.

Reply:
@pmalek You are right, this check is not needed. This is a residual check which I added initially as the schema for this  plugin changed a lot over different kong versions, this is of no use now as the schema has been added to the test itself.

Reply:
Great! This should fix it then right? #342

Reply:
No, this should fix it: #343 Else the test would fail for many of the kong versions as the schema for Acme plugin would be different.